libraries/core/src/main/kotlin/com/bumble/appyx/core/navigation/model/permanent/operation/AddUnique.kt

```diff
@@ -0,0 +1,35 @@
+package com.bumble.appyx.core.navigation.model.permanent.operation
+
+import com.bumble.appyx.core.navigation.NavElement
+import com.bumble.appyx.core.navigation.NavElements
+import com.bumble.appyx.core.navigation.NavKey
+import com.bumble.appyx.core.navigation.model.permanent.PermanentNavModel
+import kotlinx.parcelize.Parcelize
+import kotlinx.parcelize.RawValue
+
+@Parcelize
+data class AddUnique<NavTarget : Any>(
```

Comment:
Is this expected to be called by library users? otherwise internal? if it should be public, add some Javadoc? Maybe add some either way?

Reply:
I think it is more useful than regular `Add`.